server/PositivePixelCount/PositivePixelCount.py

```diff
@@ -0,0 +1,103 @@
+import os
+import sys
+
+from ctk_cli import CLIArgumentParser
+from dask import delayed
+from dask.distributed import Client
+import large_image
+import numpy as np
+import skimage.io
+
+sys.path.append(os.path.normpath(os.path.join(os.path.dirname(__file__), '..')))
+from cli_common import utils  # noqa
+
+
+def main(args):
+    Client(args.scheduler_address or None)
+    ts = large_image.getTileSource(args.inputImageFile)
+    kwargs = dict(format=large_image.tilesource.TILE_FORMAT_NUMPY)
+    makeLabelImage = args.outputLabelImage is not None
+    kwargs.update(utils.get_region_dict(
+        args.region,
+        *(args.maxRegionSize, ts) if makeLabelImage else ()
+    ))
+    if makeLabelImage:
+        tile = ts.getRegion(**kwargs)[0]
+        results, labelImage = positive_pixel_count_single_tile(args, tile, makeLabelImage=True)
+        skimage.io.imsave(args.outputLabelImage, labelImage)
+    else:
+        results = []
+        total_tiles = ts.getSingleTile(**kwargs)['iterator_range']['position']
+        for position in range(0, total_tiles, args.tile_grouping):
+            results.append(delayed(positive_pixel_count_tiles)(
+                args, kwargs, position, min(args.tile_grouping, total_tiles - position)))
+        results = delayed(combine)(results).compute()
+
+    total_all_positive = results[:3].sum()
+    output = (zip(['NumberWeakPositive', 'NumberPositive', 'NumberStrongPositive',
+                   'IntensitySumWeakPositive', 'IntensitySumPositive', 'IntensitySumStrongPositive'],
+                  results)
+              + [('IntensityAverage', results[3:].sum() / total_all_positive),
+                 ('RatioStrongToTotal', results[2] / total_all_positive),
+                 ('IntensityAverageWeakAndPositive', results[3:5].sum() / results[:2].sum())])
+    with open(args.returnParameterFile, 'w') as f:
+        for k, v in output:
+            f.write('{} = {}\n'.format(k, v))
+
+
+def combine(results):
+    return np.sum(results, 0) if results else np.zeros(6)
+
+
+def positive_pixel_count_tiles(args, kwargs, position, count):
+    ts = large_image.getTileSource(args.inputImageFile)
+    total = np.zeros(6) # In order: #weak, #+ve, #strong, sum_weak, sum_+ve, sum_strong
+    for pos in range(position, position + count):
+        tile = ts.getSingleTile(tile_position=pos, **kwargs)['tile']
+        total += positive_pixel_count_single_tile(args, tile, makeLabelImage=False)
+    return total
+
+
+def positive_pixel_count_single_tile(args, tile, makeLabelImage):
+    tile = tile[..., :3]
+    total = np.empty(6)
+    tile_hsi = rgb_to_hsi(tile / 255.)
+    mask_all_positive = (
+        (np.abs((tile_hsi[..., 0] - args.hueValue + 0.5 % 1) - 0.5) <= args.hueWidth / 2.)
+        & (tile_hsi[..., 1] >= args.saturationMinimum)
+        & (tile_hsi[..., 2] < args.intensityUpperLimit)
+        & (tile_hsi[..., 2] >= args.intensityLowerLimit)
```

Comment:
Also, this (with the `&`s leading the lines instead of ending them) runs afoul of Girder's flake8 configuration.